en/docs/basics/08_nn_graph.md

```diff
@@ -265,115 +253,116 @@ class LinearTrainGraph(flow.nn.Graph):
-       requires_grad=True))
-(OUTPUT:_model.0-output_0:tensor(..., device='cuda:0', is_lazy='True', size=(20, 1),
-       dtype=oneflow.float32))
-(MODULE:model.1:Flatten(start_dim=0, end_dim=1))
-(INPUT:_model.1-input_0:tensor(..., device='cuda:0', is_lazy='True', size=(20, 1),
-       dtype=oneflow.float32))
-(OUTPUT:_model.1-output_0:tensor(..., device='cuda:0', is_lazy='True', size=(20,), dtype=oneflow.float32))
-(OUTPUT:_model-output_0:tensor(..., device='cuda:0', is_lazy='True', size=(20,), dtype=oneflow.float32))
-(MODULE:loss_fn:MSELoss())
-(INPUT:_loss_fn-input_0:tensor(..., device='cuda:0', is_lazy='True', size=(20,), dtype=oneflow.float32))
-(INPUT:_loss_fn-input_1:tensor(..., device='cuda:0', is_lazy='True', size=(20,), dtype=oneflow.float32))
-(OUTPUT:_loss_fn-output_0:tensor(..., device='cuda:0', is_lazy='True', size=(), dtype=oneflow.float32))
-(OUTPUT:_LinearTrainGraph_0-output_0:tensor(..., device='cuda:0', is_lazy='True', size=(), dtype=oneflow.float32))
-(GRAPH:LinearTrainGraph_0:LinearTrainGraph) end building forward graph.
-(GRAPH:LinearTrainGraph_0:LinearTrainGraph) start compiling and init graph runtime.
-(GRAPH:LinearTrainGraph_0:LinearTrainGraph) end compiling and init graph rumtime.
+(GRAPH:GraphMobileNetV2_0:GraphMobileNetV2) end building graph.
+(GRAPH:GraphMobileNetV2_0:GraphMobileNetV2) start compiling plan and init graph runtime.
+(GRAPH:GraphMobileNetV2_0:GraphMobileNetV2) end compiling plan and init graph rumtime.
 ```
 
-It displays the names of the layers in the computation graph and input/output tensor information, including shape, device information, data type, and so on.
-
 The advantage of using `debug`  is that the debug information is composed and printed at the same time, which makes it easy to find the problem if there is any error in the graph building process.
 
+The Level of detail of `debug` 's output can be adjusted by setting the `v_level` parameter:
+
+```python
+graph_mobile_net_v2.debug(v_level=1)  # 输出详细信息
+```
 
 In addition to the methods described above, getting the parameters of the gradient during the training process, accessing to the learning rate and other functions are also under development and will come up soon.
 
+### Save and Load of Graph
+
+Graph reuses the network parameters of the Module, so Graph does not have its own `save` and `load` interfaces, just use the Module interface directly. You can refer to [Model saving and loading](./07_model_load_save.md).
+
+If you want to save the training results of the above `graph_mobile_net_v2`, you should actually save its Module (`model` obtained by `model = flowvision.models.mobilenet_v2().to(DEVICE)` before)
+
+```python
+flow.save(model.state_dict(), "./graph_model")
+```
+
+!!! Note
+
+    **Don't** save in the following way. Because Graph will process members when it is initialized, `graph_mobile_net_v2.model` is actually no longer a Module type:
```

Comment:
```suggestion
    **Don't** save in the following way. Because Graph will process members when it is initialized, and `graph_mobile_net_v2.model` is actually no longer a Module type:
```